lldb/test/API/functionalities/progress_reporting/TestProgressReporting.py

```diff
@@ -37,5 +37,8 @@ def test_dwarf_symbol_loading_progress_report_structured_data(self):
 
         event = lldbutil.fetch_next_event(self, self.listener, self.broadcaster)
         progress_data = lldb.SBDebugger.GetProgressDataFromEvent(event)
-        message = progress_data.GetValueForKey("message").GetStringValue(100)
-        self.assertGreater(len(message), 0)
+        title = progress_data.GetValueForKey("title").GetStringValue(100)
+        self.assertGreater(len(title), 0)
+
+        is_discrete = progress_data.GetValueForKey("is_discrete")
+        self.assertTrue(is_discrete, "is_discrete should be true")
```

Comment:
The assertion message isn't very helpful or actionable. I would probably do something like:
```
Progress event data is missing `is_discrete` key.
```

Reply:
Actually, you could check if the dictionary has the key `is_discrete` AND if its value is `true`.

Reply:
Good point, I'll update the test to add this.